MusicNotationKit/MusicNotationKit/Clef.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Clef.swift
+//  MusicNotationKit
+//
+//  Created by Kyle Sherman on 10/16/16.
+//  Copyright © 2016 Kyle Sherman. All rights reserved.
+//
+
+public struct Clef {
+
+    public let tone: Tone?
+    public let lineNumber: Double // Could be changed to Float. Don't need that precision.
+
+    /**
+     You can create a custom clef by providing a tone and line number.
+     
+     - parameter tone: The tone that the clef represents
+     - parameter lineNumber: The number representing the location on the staff. The `lineNumber`
+        starts at 0 for the first line on the staff. All spaces are the number of the preceding
```

Comment:
Lines on the staff are conventionally numbered from the bottom up: http://www.theoryinabox.org/tour/05a.html